### ISSUE TITLE
Weights are not saving

### ISSUE THREAD

**Issue by RenWild** [minimaxir/textgenrnn#33]
Also how can I save weights to train more? When I just start it like this
from textgenrnn import textgenrnn
t = textgenrnn('textgenrnn_weights.hdf5')
t.train_from_file('input.txt', num_epochs=1);
Loss will be reset again (it was 0.6 for example, and now again 1.5)

**Comment by ihavetoomanyquestions** [comment]
Not sure but did you tried new_model = False in config.yml? (i suppose it doesnt reset the weights if you do that?)

**Comment by RenWild** [comment]
Where can I find config.yml?

**Comment by minimaxir** [comment]
There should be a physical file `textgenrnn_weights.hdf5` after each epoch in the directory where the script is run. Is that the case?

**Comment by RenWild** [comment]
I am not sure about after each epoch but I have 1 textgenrnn_weights.hdf5 there.

**Comment by halesyy** [comment]
I'm getting the same issue right now, I'm using a model, vocab and config with the append of FadeToBlack, but when I go to feed it back into the generator, it just fails to load and generates a new textgenrnn_weights.hdf5
